pages/index.js

```diff
@@ -97,7 +97,7 @@ export default function Home() {
     <div className={styles.container}>
 
       <Head>
-        <title>Qibo Home Page</title>
+        <title>Qibo | Home Page</title>
```

Comment:
For consistency, we should put the `Qibo |` part in the global layout, and complete the rest with a variable to be set for each page.

However, we can also do this later on.

Reply:
I'm going to open a dedicated issue! Thank you.